tests/scripts/test_grant_executor_permissions.py

```diff
@@ -4,8 +4,7 @@
 from utils import lido
 
 
-def test_grant_executor_permissions(accounts):
-    lido_contracts = lido.contracts(network=brownie.network.show_active())
```

Comment:
unused lido imports in files after switching to fixture

Reply:
fixed